test/test_tensordict.py

```diff
@@ -80,6 +81,23 @@ def test_stack(device):
     assert (td_reconstruct == td).all()
 
 
+@pytest.mark.parametrize("device", get_available_devices())
+def test_pad(device):
+    torch.manual_seed(1)
+    dim0_left, dim0_right, dim1_left, dim1_right = [0, 1, 0, 2]
```

Comment:
Can we test with other paddings too? Like `[0, 1, 0, 2]`, `[1, 0, 0, 2]`, `[1, 0, 2, 1]` or similar?
Just to make sure that nothing goes unnoticed.